### PR TITLE
Th2 1211

### PR DESCRIPTION
**Pull request by Nikita-Smirnov-Exactpro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,4 +36,11 @@ spec:
       requests:
         memory: 100Mi
         cpu: 20m
-'''
+```
+
+# Common features
+
+This list of supported features which provided libraries.
```
Comment (ivandruzhinin) [inline README.md:43]: Did you mean "This is a list of supported features which this library provides" or "This is a list of supported features provided by libraries"? I guess it is the latter one

File: README.md [hunk 2]
```diff
@@ -36,4 +36,11 @@ spec:
...
+1. CradleMaxEventBatchSize - the option defines the maximum of the event batch size in bytes.
```
Comment (jean-michel-perli-sonsin) [inline README.md:44]: CradleMaxEventBatchSize - **this** option defines the **maximum event batch** size in bytes.

File: README.md [hunk 3]
```diff
@@ -36,4 +36,11 @@ spec:
...
+Please see more details about the future via [link](https://github.com/th2-net/th2-common-j#configuration-formats)
```
Comment (jean-michel-perli-sonsin) [inline README.md:45]: Please see more details about **this feature** via [link](https://github.com/th2-net/th2-common-j#configuration-formats)

**Review by jean-michel-perli-sonsin — CHANGES_REQUESTED** [review]
I reviewed some errors in two phrases, please take a look and correct them. Thanks.

**Review by OptimumOpium — DISMISSED** [review]
Looks fine to me. But have to update the store-common project as well. Otherwise, people will continue to create events in  the old way

**Review by OptimumOpium — APPROVED** [review]

**Pull request merged**